screens/MessagesUsers/index.js

```diff
@@ -28,6 +36,42 @@ const MessagesUsers = ({navigation}) => {
     return () => clearTimeout(timer);
   }, []);
 
+  // eslint-disable-next-line react/no-unstable-nested-components
+  const QuickActions = qaItem => {
+    return (
+      <View style={styles.qaContainer}>
+        <View style={styles.button}>
+          <Pressable onPress={() => deleteItem(qaItem.id)}>
+            <View style={styles.buttonText}>
+              <DeleteIcon width={30} height={30} fill={theme.colors.danger} />
+            </View>
+          </Pressable>
+        </View>
+      </View>
+    );
+  };
+
+  useEffect(() => {
+    setModalVisible(false);
+    if (isAnswer === 'YES') {
+      console.log(removeId);
```

Comment:
remove

Reply:
I also changed it to a more understandable name here.

Reply:
For the entire code, I will think of a more correct solution and remove all eslint related comments.